src/validation.h

```diff
@@ -659,11 +718,7 @@ CChain& ChainActive();
 /** @returns the global block index map. */
 BlockMap& BlockIndex();
 
-/** Global variable that points to the coins database (protected by cs_main) */
-extern std::unique_ptr<CCoinsViewDB> pcoinsdbview;
-
-/** Global variable that points to the active CCoinsView (protected by cs_main) */
```

Comment:
Why are you removing the comment that this data is protected by cs_main?

Reply:
Good point - since it was easy to do, I've tacked on a commit that adds lock annotations to the equivalent CoinsViews members.